test/fixtures/ambulance.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "model": "Ambulance",
+    "data": {
+      "id": "7f666fe4-dbdd-4c7f-ab44-d9157379a680",
+      "EmergencyMedicalServiceProviderId": "aac13870-f6f3-11ea-adc1-0242ac120002",
```

Comment:
@francisli Is there an easy way to ensure that the `EmergencyMedicalServiceProvider` record exists before describing this association? I hit a foreign key constraint error when the test hook runs. I know we could always manually declare the order fixtures are loaded in the `helper.js` file, but curious if `sequelize-fixtures` has anything special to handle this. I skimmed the docs for a bit, and it didn't look like it.

To get around this in the `PatientDelivery` unit test, I ended up just creating the required models in the test itself. If there's no magic silver bullet here, I'll stick with that approach and delete the json fixtures I created for it.

Reply:
You need to load the fixtures in the right order, which I don't think should be too big of an issue unless there are circular references in the tables...?

Reply:
I've changed the test to load the fixtures in the appropriate order, and use references from records in the fixtures. It gets to be a pain over time to have to manually create everything in every test, and it clutters the test code... creating a representative set of test fixtures over time will be really critical in ensuring test coverage...